index.html

```diff
@@ -1,8 +1,14 @@
 <!doctype html>
-<html lang="en" class="bg-background">
+<html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
+    <meta name="msapplication-TileColor" content="#da532c" />
+    <meta name="theme-color" content="#ffffff" />
```

Comment:
I see every case covered, even Windows 8 😅 Have you found these somewhere in documentation / article or did you google each link / meta yourself?

Reply:
I've just used favicon generator where i've uploaded single svg file only :)